tests/ports/psoc6/hw_ext/multi_stub/spi_master.py

```diff
@@ -21,6 +21,15 @@
     mosi_master_pin = "P9_0"
     miso_master_pin = "P9_1"
     ssel_master_pin = "P9_3"
+elif "CY8CKIT-062S2-AI" in board:
+    # Current not supported as
+    # SPI Slave on P9_x pins seems to be not supported
```

Comment:
Only master can be configured on this board is it?

Reply:
We could further explore this case, but connecting 9.x with 9.x does not work . I will disable the slave for this board (which I forgot). And we can investigate in the future potential spi slave configurations or?

Reply:
Disabled now slave SPI in AI KIT.